packages/kbn-unified-field-list/src/components/field_popover/field_popover_header.tsx

```diff
@@ -27,10 +27,12 @@ import type { AddFieldFilterHandler } from '../../types';
 export interface FieldPopoverHeaderProps {
   field: DataViewField;
   closePopover: EuiPopoverProps['closePopover'];
+  buttonAddBreakdownFieldProps?: Partial<EuiButtonIconProps>;
```

Comment:
**nit**: I couldn't find any usage for this property on the consuming components, is it a leftover from previous implementations?